Northstar.Client/mod/scripts/vscripts/ui/menu_ns_serverbrowser.nut

```diff
@@ -1161,10 +1162,10 @@ int function ServerSortLogic ( serverStruct a, serverStruct b )
 			bTemp = Localize( b.serverGamemode ).tolower()
 			direction = filterDirection.serverGamemode
 			break;
-		case sortingBy.LATENCY:
-			aTemp = a.serverLatency
-			bTemp = b.serverLatency
-			direction = filterDirection.serverLatency
+		case sortingBy.REGION:
+			aTemp = a.serverRegion
+			bTemp = b.serverRegion
```

Comment:
Something more should be done for sorting by region imo, like prioritising the region that the client is currently in at the top of the list instead of purely alphabetically sorting regions.

Reply:
I agree but that requires significantly more work as we'd need to determine the client's own region first, so it should be done in a separate PR, IMO.

Reply:
Potentially an easier solution would be to add a filter into the menu on the bottom or something? But yeah probably deserves it's own PR

Reply:
> client's own region first

I can add that as a HTTP header for /client/servers.

Reply:
> > client's own region first
> 
> I can add that as a HTTP header for /client/servers.

That probably could just be in launcher code if we do decide to implement it